css/styles.css

```diff
@@ -0,0 +1,311 @@
+/* Mobile css */
+.home-page {
+  display: flex;
+  flex-direction: column;
+  margin: initial;
+  padding-top: 100px;
+  background-color: #ececec;
+}
+
+.hello {
+  margin-bottom: initial;
+  margin-top: 40px;
+  color: #ec5242;
+  font-size: 20px;
+  font-weight: 500;
+  margin-left: 3vw;
+}
+
+.web-title {
+  font-weight: 900;
+  text-transform: uppercase;
+  margin-top: 5px;
+  color: #ec5242;
+  margin-left: 3vw;
+  margin-bottom: initial;
+}
+
+.h-content {
+  width: 95%;
+  margin-left: 2%;
+  margin-right: 2%;
+  border: 3px solid #fff;
+  margin-top: 20px;
+}
+
+.h-content p {
+  padding: 15px 15px;
+  margin: initial;
+  color: #272a31;
+  font-size: 14px;
+}
+
+.dates {
+  margin-left: 3vw;
+  color: #272a31;
+  margin-bottom: initial;
+  margin-top: 15px;
+}
+
+.h-copyright {
+  margin-left: 3vw;
+  margin-top: 5px;
+  margin-bottom: 60px;
+  color: #272a31;
+}
+
+/* For the main program */
+
+.main-program {
+  /* background-color: #272a31; */
+  background: url(../images/bgr.png);
+  margin-top: -20px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.p-title {
+  margin-top: 20px;
+  margin-bottom: 15px;
+}
+
+.main-program h2 {
+  margin-top: 5px;
+  color: #d3d3d3;
+  margin-bottom: 5px;
+}
+
+.line {
+  width: 40px;
+  border: 0.5px solid #ec5242;
+}
+
+.programs {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
```

Comment:
- [OPTIONAL] Kindly note that elements in the Main Programme section overlap on the mobile screen and it makes the UI not look so good. It would be nice to fix this. 👍 😎 

![image](https://user-images.githubusercontent.com/77106962/225457607-809493e8-920c-4d7c-9bed-5c7275bf8734.png)